public/js/controllers/traffic_graphs_controller.js

```diff
@@ -210,6 +210,10 @@ export class TrafficGraphsController {
         let mno_color_scheme = [],
             operators_with_color_identity = Object.keys(MNOColors);
 
+        let difference = operators.filter(x => !operators_with_color_identity.includes(x))[0];
```

Comment:
what do you mean by difference?

Reply:
Operators in array 'operators' that are not in array 'operators_with_color_identity'

Reply:
Thanks, operators_without_color_identity would be more appropriate.

Reply:
Updated thanks